### ISSUE TITLE
Default User Role shows only built-in roles.

### ISSUE THREAD

**Issue by AbortRetryFail** [Graylog2/graylog2-server#5000]
This makes group-based access to Graylog very difficult to secure. Configuring AD auth, only the built-in groups "Reader" and "Administrator" are available to select for the required "Default User Role" field.

Basically that means any user who can log in via AD gets created with at least "Reader" access, even if more restrictive roles are created and assigned by groups. For example, a role with only access to a specific dashboard.

## Expected Behavior
I would expect to be able to use custom roles in this field.

## Current Behavior
Only "Administrator" or "Reader" are available for selection.

## Possible Solution
Either allow administrators to use custom created roles as the "Default User Role" or have the option to disable automatic role assignment and rely entirely on group membership (Perhaps add a built-in "No access" role).

## Steps to Reproduce (for bugs)
1. Create a custom role
2. Configure AD authentication
3. Observe that the custom role created in step 1 is not available in the "Default User Role" pick list.

* Graylog Version: 2.4.6
* Elasticsearch Version: 5.6.10
* MongoDB Version: 3.6.6
* Operating System: CentOS 7.5
* Browser version: Chrome 67

**Comment by radykal-com** [comment]
I would like to help with this issue as soon a maintainer decides what's the best approach to this. Allowing user created roles in the "Default User Role" field is the easiest fix as only frontend changes are required

**Comment by jalogisch** [comment]
the initial thought is that the drop-down for the default role should include all present roles in the system - so all custom roles should be selectable. 

But we consider this as a feature not a bug as we might rework/rethink the integration.

**Comment by radykal-com** [comment]
@jalogisch should I open the PR with the frontend changes? or wait for further discussion?

**Comment by AbortRetryFail** [comment]
@radykal-com - I was able to change the Default User Role via the API, but it looks like it's not having the intended effect.
Even setting it to a role with only a single (empty) dashboard, Reader role gets assigned as well and the created user has access to see things like Inputs, Nodes, Overview, etc.

I was able to work around it using the User Search Pattern and the memberof:1.2.840.113556.1.4.1941: property. It's not ideal, but it works for my use case.

Microsoft docs on that are here, in case anyone else needs it: https://docs.microsoft.com/en-us/windows/desktop/adsi/search-filter-syntax
